frontend/src/pages/ArtifactList.test.tsx

```diff
@@ -14,24 +14,24 @@
  * limitations under the License.
  */
 
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 import * as React from 'react';
 import { MemoryRouter } from 'react-router-dom';
 import { Api } from 'src/mlmd/library';
 import {
   Artifact,
   ArtifactType,
+  GetArtifactsRequest,
   GetArtifactsResponse,
   GetArtifactTypesResponse,
   Value,
 } from 'src/third_party/mlmd';
-import { RoutePage } from '../components/Router';
-import TestUtils from '../TestUtils';
-import { ArtifactList } from './ArtifactList';
-import { PageProps } from './Page';
+import { ListOperationOptions } from 'src/third_party/mlmd/generated/ml_metadata/proto/metadata_store_pb';
+import { RoutePage } from 'src/components/Router';
+import TestUtils from 'src/TestUtils';
+import { ArtifactList } from 'src/pages/ArtifactList';
+import { PageProps } from 'src/pages/Page';
 
-const pipelineName = 'pipeline1';
-const artifactName = 'artifact1';
 describe('ArtifactList', () => {
```

Comment:
NIT: Consider adding `testBestPractise()` to clean up test environment before each test.